### PR TITLE
Support for Vial and update QMK to remove redundant options.

### PR DESCRIPTION
**Pull request by CountParadox**
Additionally adding some CGI.

### CODE REVIEW

**Comment by djvs** [comment]
Is this ready to merge?  Let me know

File: firmware/aek/config.h [hunk 1]
```diff
@@ -23,8 +23,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define VENDOR_ID    0x1209
 #define PRODUCT_ID   0x9209
 #define DEVICE_VER   0x0001
-#define MANUFACTURER djvs
-#define PRODUCT      aek
+#define MANUFACTURER ALPS
+#define PRODUCT      Apple Extended Keyboard 2
```
Comment (djvs) [inline firmware/aek/config.h:27]: The product/device ID correspond to a USB vendor & PID I obtained (1209 9209) and are specific to the board, please don't change these values.

File: firmware/aek/aek.h [hunk 2]
```diff
@@ -30,7 +30,7 @@
 #define ____ KC_NO
 
 #define LAYOUT( \
-    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, \
+    K000,    K001, K002, K003, K004,   K005, K006, K007, K008,   K009, \
```
Comment (djvs) [inline firmware/aek/aek.h:33]: Bunch of extra spaces?

Comment (CountParadox) [reply firmware/aek/aek.h:33]: It was the start of cleaning it up to better represent the layour, but I ended up going a different route

Comment (CountParadox) [reply firmware/aek/aek.h]: removed extra spaces

File: firmware/aek/config.h (hunk 1 above)
Comment (CountParadox) [reply firmware/aek/config.h]: reverted changes

**Comment by CountParadox** [comment]
This is now ready to merge

**Pull request merged**